addons/account_peppol/models/account_move.py

```diff
@@ -70,3 +71,18 @@ def _compute_peppol_move_state(self):
                 move.peppol_move_state = False
             else:
                 move.peppol_move_state = move.peppol_move_state
+
+    def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, model_description=False, force_email_company=False, force_email_lang=False):
+        render_context = super()._notify_by_email_prepare_rendering_context(
+            message, msg_vals, model_description=model_description,
+            force_email_company=force_email_company, force_email_lang=force_email_lang
+        )
+        if model_description == 'Invoice':
```

Comment:
This is normally translated, so I suppose it is going to be broken, isn't it ?

Reply:
model description being "Invoice" is probably the same as considering self._name == "account.move", which is always the case as you override account_move model, isn't it ?